### PR TITLE
Update Site Icons

### LINKED ISSUE #95: Icon Differentiation
Users have reported that it's difficult to differentiate the site icons on the map. Choose new (color-blind friendly) colors. Also make the icons slightly larger at the street zoom level.

### PR DESCRIPTION
**Pull request by SofieDunt**
## Why

Closes #95.

Makes the site icons more distinguishable. Also extracts styling logic from mapView to continue cleaning up the component and making it easier to unit test in the future.

## This PR

Created another color-blind friendly palette with more distinguishable colors and updated the icons to use these colors. Also made the icons different shapes and adjusted icon sizes and the street zoom level.
The styling functions check for visibility before determining the style of each feature; if the feature is invisible, just sets visibility.

## Screenshots

 
image
image
image

## Verification Steps

 
Made sure styling looked reasonable on the map.

### CODE REVIEW

File: src/components/mapPageComponents/logic/style.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  ADOPTED_ICONS,
+  OPEN_ICONS,
+  STANDARD_ICONS,
+  YOUNG_ICONS,
+} from '../../../assets/images/siteIcons';
+
+// Logic to style data layers on map
+
+/**
+ * Sets the style of the private streets layer
```
Comment (rymaju) [inline src/components/mapPageComponents/logic/style.ts:20]: Tsdoc 😍

File: src/components/mapPageComponents/mapView/index.tsx [hunk 2]
```diff
@@ -293,130 +298,129 @@ const MapView: React.FC<MapViewProps> = ({
...
+                  break;
+              }
+            });
+          }
 
-        handleZoomChange();
-      });
+          handleZoomChange();
+        })
+        .catch((err) => message.error(err.message));
```
Comment (rymaju) [inline src/components/mapPageComponents/mapView/index.tsx:411]: nice 👍

File: src/components/mapPageComponents/mapView/index.tsx [hunk 3]
```diff
@@ -113,165 +110,173 @@ const MapView: React.FC<MapViewProps> = ({
...
-          mapTypeControl: false,
-          // mapId: mapId,
-          restriction: {
-            latLngBounds: BOSTON_BOUNDS,
-            strictBounds: false,
-          },
-        });
+      LOADER.load()
+        .then(() => {
+          map = new google.maps.Map(mapElement, {
```
Comment (rymaju) [inline src/components/mapPageComponents/mapView/index.tsx:115]: Theres _SO_ much code in this anonymous function 🤯 . We really need to pull most of this out. I could barely even find the .catch

File: src/components/mapPageComponents/mapView/index.tsx [hunk 4]
```diff
@@ -293,130 +298,129 @@ const MapView: React.FC<MapViewProps> = ({
...
+                  },
+                });
+              },
+              () => {
+                message
+                  .info(
+                    'Enable access to your location to display where you are on the map.',
+                    5,
+                  )
+                  .then();
```
Comment (rymaju) [inline src/components/mapPageComponents/mapView/index.tsx:374]: Does this then() do anything? 🤔

File: src/components/mapPageComponents/mapView/index.tsx [hunk 5]
```diff
@@ -113,165 +110,173 @@ const MapView: React.FC<MapViewProps> = ({
...
-          },
-        });
+      LOADER.load()
+        .then(() => {
+          map = new google.maps.Map(mapElement, {
+            center: BOSTON,
+            zoom: 12,
+            fullscreenControl: false,
+            mapTypeControl: false,
+            // mapId: mapId,
```
Comment (rymaju) [inline src/components/mapPageComponents/mapView/index.tsx:120]: ❓  Should this stay commented out for any reason or can we delete this?

**Review by rymaju — APPROVED** [review]
Some nitpicks. Nothing too blocking. Overall looks solid, wonderful refactoring 🌈

File: src/components/mapPageComponents/mapView/index.tsx (hunk 3 above)
Comment (SofieDunt) [reply src/components/mapPageComponents/mapView/index.tsx:115]: I could barely find the place to put the catch 😂 . The next ticket for the map is pulling everything out and testing

File: src/components/mapPageComponents/mapView/index.tsx (hunk 4 above)
Comment (SofieDunt) [reply src/components/mapPageComponents/mapView/index.tsx:374]: Not really, it just gets rid of the warning that the promise from the message is ignored

File: src/components/mapPageComponents/mapView/index.tsx (hunk 5 above)
Comment (SofieDunt) [reply src/components/mapPageComponents/mapView/index.tsx:120]: It doesn't seem like there's a reason to me, will look into it/take it out in the map cleanup ticket

**Pull request merged**